index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title id="title">Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="css/styles.css">
+      </head>
+
+  <body>
+    <main id="main">
+      <h1><center>Dr. Norman Borlaug</center></h1>
+      <p><center>The man who saved a billion lives</center></p>
+      <div id="img-div">
+        <center><img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" ></center>
+        <p id="img-caption"><center>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</center></p>
+      </div>
+      <p id="tribute-info">Here's a time line of <a id="tribute-link" target="_blank" href="#">Dr. Borlaug's</a> life:</p>
+     <div>
+      <ul>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+        <li>1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>1914 - Born in Cresco, Iowa</li>
+      </ul>
```

Comment:
I think your list will look much better after being aligned properly as per the original document.